git/README.md

```diff
@@ -7,24 +7,53 @@ OCA maintainer quality tools.
 
 For using it, you can choose from two options:
 
-1. Copy files to local git repository into .git/hooks, and you will enable the
+1. Create symbolic link of files to local git repository into .git/hooks, and you will enable the
    checks for that git repository.
 
-2. Copy files to git template directory. In Ubuntu, it's located in:
+   ```bash
+   git clone git@github.com:OCA/maintainer-quality-tools.git ${HOME}/lint-hook
+   ln -sf ${HOME}/lint-hook/git/* {YOUR_PROJECT}/.git/hooks/.
+   ```
+
+2. Create symbolic link of files to git template directory.
+
+  In Ubuntu, it's located in:
+
+     /usr/share/git-core/templates/hooks/
+
+
+  In MAC, it's located in:
+
+    Installed from MAC-port
+
+        /opt/local/share/git-core/templates/hooks/
 
-   /usr/share/git-core/templates/hooks/
 
 Then, making `git init` inside a git repository, these files will be copied,
```

Comment:
@osvalr 
Good comment with a `install hook`.
Now is too much for this PR that have a focus: `compatibility with python migration`
But you can create other one (progressive maybe) with this feature.